x11-misc/evolvotron/evolvotron-0.6.3-r1.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit qmake-utils
+
+DESCRIPTION="Generative art image evolver"
+HOMEPAGE="https://sourceforge.net/projects/evolvotron/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~x86-fbsd"
+IUSE=""
+
+RDEPEND="
+	dev-libs/boost
+	dev-qt/qtcore:4
+	dev-qt/qtgui:4
+"
+DEPEND="${RDEPEND}"
+
+S=${WORKDIR}/${PN}
+
+PATCHES=( "${FILESDIR}"/${P}-gcc6.patch )
+
+DOCS=( BUGS NEWS README TODO USAGE )
```

Comment:
is `USAGE` that important? Does it contain important instructions? If not, I would just nix `DOCS` (your decision though)